### ISSUE TITLE
is the docker published by the author?

### ISSUE THREAD

**Issue by 2236529177** [abyzovlab/CNVnator#218]
hi, thanks for making such a poweful tool, 
due to the install is too hard, I tried to find a docker vesion, but in the github, you never mentioned this, so does diploid/cnvnator made by you, if so, is there any guide tell me how to use, 
 docker pull diploid/cnvnator
thanks a lot

**Comment by suvakov** [comment]
No, none of existing docker images were made by Abyzov lab. You can find [Dockerfile](https://github.com/abyzovlab/CNVnator/blob/master/Dockerfile) in the repository made by external contributor @nathanweeks. We are switching development to CNVpytor, Python impemented successor of CNVnator. We expect docker image for CNVpytor soon. You can follow the progress on [CNVpytor Github page](https://github.com/abyzovlab/CNVpytor).
